terraform/aks/variables.tf

```diff
@@ -151,7 +151,12 @@ locals {
 
   app_env_values = merge(
     local.app_config,
-    { DB_SSLMODE = local.postgres_ssl_mode }
+    {
+      DB_SSLMODE = local.postgres_ssl_mode
+      BIGQUERY_PROJECT_ID = "teaching-qualifications",
```

Comment:
@gpeng - I wasn't sure whether to add these here using `var.environment` orrrr here: https://github.com/DFE-Digital/check-childrens-barred-list/blob/main/terraform/aks/config/app_config.yml

If in that app_config file, it would look like this:

```
default: &default
  RAILS_SERVE_STATIC_FILES: true
  BIGQUERY_PROJECT_ID: "teaching-qualifications"
  BIGQUERY_TABLE_NAME: "events"

preproduction:
  <<: *default
  RAILS_ENV: production
  RACK_ENV: production
  BIGQUERY_DATASET: events_preprod

production:
  <<: *default
  RAILS_ENV: production
  RACK_ENV: production
  BIGQUERY_DATASET: events_production

review:
  <<: *default
  RAILS_ENV: review
  RACK_ENV: review
  BIGQUERY_DATASET: events_review

test:
  <<: *default
  RAILS_ENV: production
  RACK_ENV: production
  BIGQUERY_DATASET: events_test
```